### ISSUE TITLE
[iOS] Crash iOS 16

### ISSUE THREAD

**Issue by tatianefx** [ZupIT/beagle#1825]
Description and Example
The issue still keeps happening on iOS 16 and is related to removeSubrange for Substrings.

Reproducing:
```swift
var sub = "a"[...]
sub.removeSubrange(sub.startIndex..<sub.endIndex) 
// Swift/StringGraphemeBreaking.swift:458: 
// Fatal error: Unexpectedly found nil while unwrapping an Optional value
```

**Comment by dantes-git** [comment]
This issue has already been fixed in [1.10.3](https://github.com/ZupIT/beagle-ios/releases/tag/1.10.3) and [2.0.2](https://github.com/ZupIT/beagle-ios/releases/tag/2.0.2). So far (tested on iOS 16 rc) the substring api persists with this issue.

**Comment by Tiagoperes** [comment]
I'm closing this issue since its been already hot-fixed.
